### PR TITLE
Improve descriptive text

### PR DESCRIPTION
**Pull request by karl9000**
More comments have been added to explain the used methods. Furthermore,
the axis-argument for methods like numpy.sum is briefly introduced.

### CODE REVIEW

File: python/numeric-python.ipynb [hunk 1]
```diff
@@ -223,14 +227,36 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Nützliche Eigenschaften"
+    "# Dimension von Arrays\n",
+    "\n",
+    "In Numpy werden Dimensionen auch *Axen* genannt."
```
Comment (chrbeckm) [inline python/numeric-python.ipynb:259]: Better: *Achsen (axis)*

File: python/numeric-python.ipynb [hunk 2]
```diff
@@ -242,8 +268,8 @@
     "a = np.array([1.5, 3.0, 4.2])\n",
     "b = np.array([[1, 2], [3, 4]])\n",
     "\n",
-    "print(a.ndim, a.shape, a.size, a.dtype)\n",
-    "print(b.ndim, b.shape, b.size, b.dtype)"
+    "print('Array a\\n\\tndim\\t', a.ndim, '\\n\\tshape\\t', a.shape, '\\n\\tsize\\t', a.size, '\\n\\tdtype\\t', a.dtype)\n",
+    "print('Array b\\n\\tndim\\t', b.ndim, '\\n\\tshape\\t', b.shape, '\\n\\tsize\\t', b.size, '\\n\\tdtype\\t', b.dtype)"
```
Comment (chrbeckm) [inline python/numeric-python.ipynb:272]: `print(a)` & `print(b)` would be good.
f-strings could be an option too.

File: python/numeric-python.ipynb [hunk 3]
```diff
@@ -543,9 +570,13 @@
...
     "\n",
-    "print(a >= 1)\n",
+    "print(a)\n",
+    "\n",
+    "# create a mask for all elements >= 1\n",
+    "mask = a >= 1\n",
+    "print(mask)\n",
     "\n",
-    "print(a[a >= 1])"
+    "print(a[mask])"
```
Comment (chrbeckm) [inline python/numeric-python.ipynb:579]: Show both options?

File: python/numeric-python.ipynb [hunk 4]
```diff
@@ -584,6 +654,13 @@
     "np.prod(x)"
    ]
   },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "Mittelwert der Einträge"
+   ]
+  },
```
Comment (chrbeckm) [inline python/numeric-python.ipynb:663]: markdown cell

File: python/numeric-python.ipynb [hunk 5]
```diff
@@ -223,14 +227,36 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Das erlaubt es z.B. eine ganze Tabelle als ein Array abzuspeichern."
+    "Das erlaubt es z.B. eine ganze Tabelle als ein Array abzuspeichern.\n",
+    "\n",
+    "Bei mehrdimensionalen Arrays sind auch Matrixoperationen möglich:"
```
Comment (maxnoe) [inline python/numeric-python.ipynb:232]: Auch bei 1D (vektoren):

```python
>>> import numpy as np
>>> np.array([1, 2, 3]) @ np.array([4, 5, 6])
32

```

**Review by chrbeckm — APPROVED** [review]

**Pull request merged**